src/libraries/Microsoft.Extensions.Configuration.Binder/src/buildTransitive/Microsoft.Extensions.Configuration.Binder.targets

```diff
@@ -8,6 +8,16 @@
         </ItemGroup>
     </Target>
 
+    <Target Name="_Microsoft_Extensions_Configuration_Binder_EnableInterceptors"
+            Condition="'$(EnableConfigurationBindingGenerator)' == 'true'"
+            AfterTargets="ResolvePackageDependenciesForBuild;ResolveNuGetPackageAssets">
```

Comment:
Why do you need an AfterTargets here to just a property? That doesn't make sense.

Reply:
Do you even need a target here at all?